Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/Booking.hs

```diff
@@ -192,17 +192,33 @@ findAssignedByRiderId personId = Esq.buildDType $ do
 findAssignedByEstimateId :: Transactionable m => Id Estimate -> m (Maybe Booking)
 findAssignedByEstimateId estimateId = Esq.buildDType $ do
   fullBookingsT <- Esq.findOne' $ do
-    (booking :& fromLoc :& mbToLoc :& mbTripTerms :& mbRentalSlab :& _ :& driverOffer) <-
+    (booking :& _ :& driverOffer :& fromLoc :& mbToLoc :& mbTripTerms :& mbRentalSlab) <-
       from $
-        fullBookingTable
+        table @BookingT
           `innerJoin` table @Quote.QuoteT
-            `Esq.on` ( \(rb :& _ :& _ :& _ :& _ :& quote) ->
+            `Esq.on` ( \(rb :& quote) ->
                          rb ^. RB.BookingQuoteId ==. Esq.just (quote ^. Quote.QuoteTId)
                      )
           `innerJoin` table @DrOff.DriverOfferT
-            `Esq.on` ( \(_ :& _ :& _ :& _ :& _ :& quote :& driverOffer) ->
+            `Esq.on` ( \(_ :& quote :& driverOffer) ->
                          quote ^. Quote.QuoteDriverOfferId ==. Esq.just (driverOffer ^. DrOff.DriverOfferTId)
                      )
+          `innerJoin` table @Loc.BookingLocationT
```

Comment:
This query anyways is require to get bookingId, can we just fetch bookingId and don't join these tables.